packages/richtext-lexical/src/lexical/LexicalEditor.tsx

```diff
@@ -112,6 +107,7 @@ export const LexicalEditor: React.FC<
           }
           ErrorBoundary={LexicalErrorBoundary}
         />
+        <NormalizeSelectionPlugin />
```

Comment:
The issue was resolved with the changes above, so this isn't strictly necessary, but I decided to add it to make the editor more robust and prevent errors in the future. Additionally, I believe this will fix a bug in the email builder plugin.